src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/PrefsFramework.md

```diff
@@ -0,0 +1,775 @@
+this component as a Primary Schema with the Framework.** The Framework will automatically record
+this fact and use this Primary Schema with your Preference Editor.
+</dd>
+<dt>`schema`</dt>
+<dd>
+This property is the actual JSON definition of the preferences for this Preference Editor.
+</dd>
+</dl>
+
+In this particular example, only a single preference is being defined; a boolean called
+“minEditor.autoPilot”:
+
+```javascript
+"minEditor.autoPilot": {
+    "type": "boolean",
+    "default": false
+}
+```
+The key in the JSON definition is the name of the preference.
+This name will be used throughout the Preferences Framework to associated all the components
+related to the setting. The name can be anything, so long as it is used consistently,
+but keep in mind that it will be used in the persistent storage for the user's preference,
+and will be shared with other technologies that may wish to define enactors to respond to it.
+We recommend that it be thoughtfully namespaced and human-understandable.
+
+The value is an object containing the properties of the preference.
+Every preference in a Primary Schema must have at least two properties: `“type”` and `“default”`.
+
+_Coming soon: More information about these two properties_
+
+#### Panel ####
+
+
+A [Panel](../Panels.md) is a [component](../UnderstandingInfusionComponents.md) responsible for rendering the user interface controls for a
+preference and tying them to the internal model] that represents the preference value.
+The Panel for the auto-pilot preference control is defined in the `minEditor.js` file:
+<aside class="infusion-docs-callout">
+[Components](../UnderstandingInfusionComponents.md) are the core building-blocks of
+any Infusion application.
+An Infusion component can represent a visible component on screen, a collection of related
+functionality (such as an object, as in object-orientation), or simply a unit of work or
+relationship between other components.
+</aside>
+
+```javascript
+/**
+ * Panel for the auto-pilot preference
+ */
+fluid.defaults("minEditor.panels.autoPilot", {
+    gradeNames: ["fluid.prefs.panel"],
+
+    // the Preference Map maps the information in the primary schema to this panel
+    preferenceMap: {
+        // the key must match the name of the pref in the primary schema
+        "minEditor.autoPilot": {
+            // this key is the path into the panel's model where this preference is stored
+            "model.autoPilot": "default"
+        }
+    },
+
+    // selectors identify elements in the DOM that need to be accessed by the code;
+    // in this case, the Renderer will render data into these particular elements
+    selectors: {
+        autoPilot: ".mec-autoPilot"
+    },
+
+    // the ProtoTree is basically instructions to the Renderer
+    // the keys in the prototree match the selectors above
+    protoTree: {
+        // this value is an IoC reference to the last part of the model path in the preferenceMap
```

Comment:
The only reference in our docs just refers to an "EL path" https://github.com/fluid-project/infusion-docs/blob/master/src/documents/tutorial-gettingStartedWithInfusion/RendererComponents.md#example-currency-converter (or EL reference) at the end of the link.
On the other hand we could just call it an IoC reference and just add the note that "IoC references resolve differently within the protoTree area and in the simple form resolve onto the model of this component".